cime_config/buildnml

```diff
@@ -43,7 +43,8 @@ def _create_drv_namelists(case, infile, confdir, nmlgen, files):
     config['MPILIB'] = case.get_value('MPILIB')
     config['OS'] = case.get_value('OS')
     config['glc_nec'] = 0 if case.get_value('GLC_NEC') == 0 else case.get_value('GLC_NEC')
-    config['single_column'] = 'true' if case.get_value('PTS_MODE') else 'false'
+    if case.get_value("ATM_NX") == 1 and case.get_value("ATM_NY") == 1 and case.get_value("ATM_DOMAIN_FILE") != "UNSET":
+        config['single_column'] = 'true'
```

Comment:
We only want single_column = True for the case when you are running SCAM mode with global datasets. Turning single_column to true for cases of a single point exercises different code, that you don't need to. And it has limitations like CTSM can't restart. So we can't set it up this way. I really think that PTS_MODE needs to come back in. I like the idea of simplifying when we can, but I don't see that we can do that here.